README.md

```diff
@@ -89,6 +89,17 @@ Your `~/.vimrc.local` might look like this:
     highlight NonText guibg=#060606
     highlight Folded  guibg=#0A0A0A guifg=#9090D0
 
+If you don't wish to install a vim plugin from the default set of vim plugins in
+`.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
+your `~/.vimrc.bundles.local`.
+blacklist in `~/.vimrc.ignorebundles`:
+
+    " Don't install vim-scripts/tComment
+    UnPlug 'tComment'
+
+`UnPlug` can be used to install your own fork of a plugin or to install a shared
+plugin with different custom options.
```

Comment:
I think it could be useful to provide an example for this. Maybe two lines, one with `UnPlug` and one with `Plug` adding it back with specific options.